Sources/Networking/Responses/RevenueCatUI/PaywallComponentsData.swift

```diff
@@ -174,6 +182,13 @@ extension PaywallComponentsData {
             _revision = 0
         }
 
+        do {
+            deepLinkID = try container.decode(String.self, forKey: .deepLinkID)
+        } catch {
+            errors["deep_link_id"] = .init(error)
```

Comment:
[Not related to this PR!]
It's interesting how these errors are structured as a map. In android, I think these are a list. But I think this works as well 🤷